docs/runtime-cost.md

```diff
@@ -11,3 +11,39 @@ Here is a list of suggestion on how to measure the cost of running an AEA:
 - the cost of basic components: dialogues memory relative to number of messages, SOEF connection baseline memory usage, P2P connection baseline memory usage, smart contract baseline memory usage
 
 The `aea run --profiling SECONDS` command can be used to report measures in all of the above scenarios.
+
+## Running and profiling a locally modified AEAs
+
+In order to run a locally modified version of the framework, you will need to build the container using the Dockerfile_local at develop_image and tag it. From the open-aea root, run:
+
+```docker build . -t valory/open-aea-user:0.1.0 --file ./develop-image/Dockerfile_local```
+
+Now let's run some agents from the consensus-alorithms's repository, for example price_estimation, an oracle that aggregates bitcoin prices from different sources. This repository can also have some local modifications.
```

Comment:
This documentation should live in the consensus-algorithms repo, not here imo.

Reply:
Although it would be nice to keep it here for general usage. Hence I recommend you generalise this guide for arbitrary AEAs and remove references to the other repo